dbt/adapters/hive/connections.py

```diff
@@ -226,6 +228,10 @@ def open(cls, connection):
     def exception_handler(self, sql: str):
         try:
             yield
+        except HttpError as httpError:
+            print ("HTTP Authorization error: " + str(httpError))
```

Comment:
The httperror 'cannot unpack non-iterable object' is not very user friendly here.  
Can we just pass that to log for our debugging and  for user just print a friendly message like "HTTP  401 Authorization error: Please check you have entered correct credentials)